addons/account/data/account_data.xml

```diff
@@ -145,12 +145,6 @@
             <field name="payment_type">outbound</field>
         </record>
 
-        <!-- Account Tax Group -->
-        <record id="tax_group_taxes" model="account.tax.group">
-            <field name="name">Taxes</field>
-            <field name="sequence">0</field>
-        </record>
```

Comment:
The tax groups are now instantiated on the company instead of cross company master data.
It allows to customize the name, it is more logical when you see the list view, no more confusion with a default group that contains stuff from all countries.
All the fields that were on it were company properties, the schema didn't make sense.

It is also more logical for the declaration of the data, even though it is optional and could be reverted